jobs/build/release-branch/Jenkinsfile

```diff
@@ -0,0 +1,108 @@
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+  if(env.WORKSPACE == null) {
+    env.WORKSPACE = pwd()
+  }
+}
+
+def ssh(cmd) {
+  sh "ssh openshiftdevel -- bash -x <<'EOF'\n${cmd}\nEOF"
+}
+
+def mail_failure(err, approvers) {
+  mail(
+    to: approvers, from: "aos-cd@redhat.com",
```

Comment:
True -- it will basically just be Clayton. Approvers list should be fine for this, we can update if more people end up owning the release.